projects/playground/src/app/app.component.ts

```diff
@@ -4,9 +4,10 @@ import { ChangeDetectionStrategy, Component } from '@angular/core';
 @Component({
   selector: 'pl-root',
   template: `
-    <pl-1-plain-text></pl-1-plain-text>
-    <!--<pl-2-text-in-box></pl-2-text-in-box>-->
-    <!--<pl-3-dashboard></pl-3-dashboard>-->
+  <layout>
+    <pl-plain-text width="50%"></pl-plain-text>
```

Comment:
Also, we need to find a way to keep **Dashboard Demo** working after refactoring

Reply:
So, I don't think we can remove demos from here, also as remove blessed contrib

Reply:
But, I think we can move those demos into some kind of playground. Anyway, we still need to keep blessed-contrib library